### PR TITLE
Rafael José  Jiménez´s card

### PR DESCRIPTION
**Pull request by RafaelJoseJimenez**
added a new card and modify the content with my info

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
...
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        
+        <!-- ________ *Rafael* Contributor card START ________  -->
+      <div class="card">
+        <p class="name">Rafael José Jiménez</p>
+        <p class="contact">
+          <i class="fab fa-twitter"></i>
+          <a href="https://twitter.com/fistan85" target="_blank">@fistan85</a>
+        </p>
+        <p class="about">...</p>
```
Comment (TenTraicion) [inline index.html:125]: Kindly added about info
```suggestion
        <p class="about">Your about info here...</p>
```

File: index.html [hunk 2]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
...
+        <div class="resources">
+          <p>3 Useful Dev Resources</p>
+          <ul>
+            <li>
+              <a href="#" target="_blank" title="First Resource">Resource 1</a>
```
Comment (TenTraicion) [inline index.html:130]: Kindly added the resources
`<a href=`"Your resource URL", `title=`"Resorce Title and Put the resource name in the place of `Resource 1`

add 3 resources to complete your card

**Review by TenTraicion — CHANGES_REQUESTED** [review]
kindly do the changes and commit

**Comment by CandidDeer** [comment]
@RafaelJoseJimenez Due to two of us maintainers working simultaneously on your PR, an error has occured. I am very sorry about that and kindly ask you to create a new PR with your card info.

Let's take this as a chance to make sure your PR is perfect when opened and can be merged straight away. 👍🏻 

While you made a great start, including your name and twitter account in the card template, please make sure to also enter a **short description about who you are inside the tag <p class="about">...</p>**.

You will then need to proceed with entering the **first great resource** you'd like to share with the world inside the  **<a href="#" target="_blank" title="First Resource">Resource 1</a>** tag. The # inside the tag should be replaced with your resource's URL. Write your resource's name instead of "Resource 1".

Then proceed the same way with your **second resource** and **third resource** and you'll end up with the perfect PR to commit. 💯 

We will then make sure to commit it after review.

Please let me or my maintainer colleagues know if you have any further questions.

Happy coding! ✨

**Pull request merged**